examples/hardware_encoding/main.go

```diff
@@ -72,6 +86,8 @@ func main() {
 	hardwarePixelFormat := astiav.FindPixelFormatByName(*hardwarePixelFormatName)
 	if hardwarePixelFormat == astiav.PixelFormatNone {
 		log.Fatal("main: hardware pixel format not found")
+	} else if !slices.Contains(valid_hw_formats, hardwarePixelFormat) {
+		log.Fatal("main: hardware pixel format not supported : ", hardwarePixelFormat)
```

Comment:
Can you use `log.Fatalf("main: hardware pixel format not supported: %s", hardwarePixelFormat)` instead?